save-cloud-common/src/jvmMain/kotlin/com/saveourtool/save/entities/Execution.kt

```diff
@@ -115,15 +164,22 @@ class Execution(
             batchSize = 20,
             type = ExecutionType.GIT,
             version = null,
-            0,
-            0,
-            0,
-            0,
+            allTests = 0,
```

Comment:
this method `stub` is used in real code (not only for testing purpose)
all fields `*Tests` are not null in database and not null in model: they are set to `0` on creation. Applied the same logic for `*Checks`.

Suggestion: migrate to nullable as for `TestExecution`